x-pack/plugin/sql/sql-client/src/main/java/org/elasticsearch/xpack/sql/client/ConnectionConfiguration.java

```diff
@@ -73,6 +73,9 @@ public class ConnectionConfiguration {
     public static final String ALLOW_PARTIAL_SEARCH_RESULTS = "allow.partial.search.results";
     public static final String ALLOW_PARTIAL_SEARCH_RESULTS_DEFAULT = "false";
 
+    // CPS project routing
+    public static final String PROJECT_ROUTING = "project.routing";
```

Comment:
For my education, jdbc does not allow `_` in property names?
I am asking as it is `project_routing` everywhere else.